components/skus/browser/rs/wasm/src/lib.rs

```diff
@@ -165,8 +171,9 @@ impl JSSDK {
             let order = sdk
                 .submit_order_credentials_to_sign(&order_id)
                 .await
-                .map_err(|e| e.to_string())?;
-            Ok(JsValue::from_serde(&order).unwrap())
+                .map_err(|e| JsError::from(e))?;
```

Comment:
likewise, and all the others in this file. `cargo clippy` will find them for you.

Reply:
https://github.com/brave/brave-core/pull/13820/commits/448432a1248d454e47559d00032f42420812f015